src/coreclr/jit/lowerxarch.cpp

```diff
@@ -1912,6 +1913,7 @@ GenTree* Lowering::LowerHWIntrinsicCreate(GenTreeHWIntrinsic* node)
     var_types      simdBaseType    = node->GetSimdBaseType();
     unsigned       simdSize        = node->GetSimdSize();
     simd32_t       simd32Val       = {};
+    simd64_t       simd64Val       = {};
```

Comment:
We shouldn't need both to be declared in the same scope. We can just `simd64Val.v256[0]` to get a `simd32_t`.